src/main/java/ua/ithillel/lesson15/ProductMoving.java

```diff
@@ -0,0 +1,88 @@
+package ua.ithillel.lesson15;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class ProductMoving {
+
+    private final List<Product> list = new ArrayList<>();
+
+    public void add(Product product){
+        list.add(product);
+    }
+
+    public List<Product> getWholeList(){
+        return list;
+    }
+
+    public List<Product> getCategoryForPrice(Category category, double minPrice) {
+
+        List<Product> categoryAndPrice = list.stream()
```

Comment:
```suggestion
        return list.stream()
```
немає сенсу робити метод довшим (окрім випадку, коли треба щось подебажити - то може бути ок, але після дебагу знову повернути одразу return) 

Так само буде і для інших.